src/components/events/EventsSection.jsx

```diff
@@ -18,23 +19,29 @@ class EventsSection extends React.Component {
   }
 
   componentDidMount() {
-    db.collection('events').onSnapshot((querySnapshot) => {
-      const events = [];
-      querySnapshot.forEach((doc) => {
-        const { eventTitle, numJudges, eventDate } = doc.data();
+    const events=[];
+    axios.get('http://localhost:5000/events').then((response) => {
+      for (var key in response.data){
+        const e = response.data[key]
+        const eventTitle = e.event_title;
+        const id = e.id;
+        const numJudges = e.num_judges;
+        const eventDate = e.event_date;
+
         const date = isObject(eventDate)
-          ? new Date(eventDate.seconds * 1000).toLocaleDateString('en-GB')
-          : eventDate;
+        ? new Date(eventDate.seconds * 1000).toLocaleDateString('en-GB')
+        : eventDate;
+
         const event = {
           eventDate: date,
           eventTitle,
-          id: doc.id,
-          numJudges
-        };
+          id,
+          numJudges 
+        }
         events.push(event);
```

Comment:
once this has been changed to functional, we should be able to remove this line

Reply:
Removed in https://github.com/uwblueprint/dancefest-web/pull/94/commits/14df67de75375d7d3c265cac721d763d08a9eb69